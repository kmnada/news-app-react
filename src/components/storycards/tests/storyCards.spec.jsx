import React from 'react';
import { shallow } from 'enzyme';
import StoryCards from '../storyCards';

describe(' StoryCards component', () => {
  it('renders default component', () => {
    const wrapper = shallow(<StoryCards />);
    expect(wrapper).toMatchSnapshot();
  });
});
