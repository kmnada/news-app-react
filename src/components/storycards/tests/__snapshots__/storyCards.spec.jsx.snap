// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render StoryCards component 1`] = `ShallowWrapper {}`;
